### PR TITLE
Feature/add social media

### PR DESCRIPTION
**Pull request by AhlOct**

### CODE REVIEW

**Comment by empeje** [comment]
Please share your preview link ya

**Comment by AhlOct** [comment]
You can preview in this link : https://raw.githack.com/AhlOct/open-source/feature/Add-SocialMedia/en_ID/#!index.md

**Comment by empeje** [comment]
Good job, please exclude the changes of you adding you as an intern with changes on the social media. Currently, it is mixed together.

Hint: learn more about git branch.

**Comment by AhlOct** [comment]
I was commit for remove me on interns member at branch feature/Add-SocialMedia. I make new branch from feature/Add-AhlOct, which should be from gh-pages (master)

File: en_ID/pages/interns.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Put images link below
 [ojasthisside-img]: https://github.com/ojasthisside.png?size=25
 [fatwadwihidayat-img]: https://github.com/FatwaDwiHidayat.png?size=25
 [hananaskarim-img]: https://github.com/HananAskarim.png?size=25
-[MuhammadAzka9-img]:https://github.com/MuhammadAzka9.png?size=25
+[MuhammadAzka9-img]: https://github.com/MuhammadAzka9.png?size=25
```
Comment (empeje) [inline en_ID/pages/interns.md]: you add an extra space which is unnecessary here

Comment (AhlOct) [reply en_ID/pages/interns.md]: i have updated it

File: en_ID/pages/interns.md [hunk 2]
```diff
@@ -26,7 +26,6 @@ Put images link below
 [fatwadwihidayat-img]: https://github.com/FatwaDwiHidayat.png?size=25
 [hananaskarim-img]: https://github.com/HananAskarim.png?size=25
 [MuhammadAzka9-img]:https://github.com/MuhammadAzka9.png?size=25
-
```
Comment (empeje) [inline en_ID/pages/interns.md]: removal of this new line is also not necessary, when creating a PR we should focus on the changes required.

Comment (AhlOct) [reply en_ID/pages/interns.md]: i have returned that line

**Comment by empeje** [comment]
Nicely done @AhlOct 🎉

**Pull request merged**